sql-schema/127.sql

```diff
@@ -0,0 +1,11 @@
+DROP TABLE IF EXISTS `ipmi_sensors`;
```

Comment:
I think the rewrite is great but I'm not sure of using a new table, components or static arrays would be the best way to go.